frontend/src/metabase/collections/components/BaseItemsTable.jsx

```diff
@@ -139,6 +148,18 @@ function BaseItemsTable({
           }
         >
           <tr>
+            <ColumnHeader>
+              <BulkSelectWrapper>
+                <CheckBox
+                  checked={selectedItems?.length > 0 || false}
+                  indeterminate={
+                    (selectedItems?.length > 0 && hasUnselected) || false
+                  }
+                  onChange={hasUnselected ? onSelectAll : onSelectNone}
+                  aria-label={t`Select all items`}
```

Comment:
At this stage of the release process, we can't add new _unique_ user-facing strings. We've locked all the new strings added during the v46 cycle, and any new string won't show up for translators and hence translated. There's a "Select all" line added long time ago that we should be able to re-use

Reply:
Ah so this PR is targeting v47 instead 46, so is it okay to add this string or do I still need to reuse the old "Select all" line?

Reply:
Oh, if it's targeting 47, then it's okay to keep the new string. But make sure to merge it _only_ after the RC is cut. We'd create a `release-x.46.x` branch and build the final release for it instead of `master`. Once the branch is created it's safe to merge stuff to `master` again